Assets/Scripts/saboteurManager.cs

```diff
@@ -0,0 +1,160 @@
+                break;
+            case "sentMessage":
+                Component[] messageComponents;
+                OutputMessage clueUpdate = new OutputMessage();
+                clueUpdate.noun = message.noun;
+                persistMessage.noun = clueUpdate.noun;
+                clueUpdate.verb = message.verb;
+                persistMessage.verb = clueUpdate.verb;
+                clueUpdate.step = message.step;
+                persistMessage.step = clueUpdate.step;
+                messageComponents = GetComponents<OutputManager>();
+
+                foreach (OutputManager manager in messageComponents)
+                    manager.updateSendWindow(clueUpdate);
+                break;
+            case "updateGuess":
+                Component[] guessComponents;
+                OutputMessage guessUpdate = new OutputMessage();
+                guessUpdate.nouns = message.nouns;
+                guessUpdate.verbs = message.verbs;
+                guessUpdate.role = message.role;
+                guessComponents = GetComponents<OutputManager>();
+
+                foreach (OutputManager manager in guessComponents)
+                    manager.updateGuessWindow(guessUpdate);
+                break;
+            case "correctGuesses":
+                Component[] correctGuessComponents;
+                OutputMessage correctGuessUpdate = new OutputMessage();
+                correctGuessUpdate.correctGuesses = message.correctGuesses;
+                break;
+            case "gameOver":
+                break;
+            default:
+                Debug.Log("No matching event found for scene...");
+                break;
+        }
+    }
+
+    public void send_guess()
+    {
+        for(int i = 0; i < 4; i++)
+        {
+            guess_message.verbs[i] = verb_Guess[i].captionText.text.ToString();
+            guess_message.nouns[i] = noun_Guess[i].captionText.text.ToString();
+        }
+        guess_message.playerId = playerState.playerId;
+        guess_message.sessionId = playerState.sessionId;
+
+        guess_list.Add(guess_message);
+        
+        try
+        {
+            NetworkMessaging.SendJsonViaPOST(guess_message, "http://localhost:8095/sendmessagedemo");
+        }
+        catch (SystemException e)
+        {
+            Debug.Log(e.Message.ToString());
+        }
+    }
+
+    public void send_saboteur_message()
+    {
+        next_message.step = persistMessage.step;
+        next_message.noun = (noun_dropdown.captionText.text.ToString());
+        if(next_message.noun == "") next_message.noun = persistMessage.noun;
+        next_message.verb = (verb_dropdown.captionText.text.ToString());
+        if(next_message.verb == "") next_message.verb = persistMessage.verb;
+        next_message.playerId = playerState.playerId;
+        next_message.sessionId = playerState.sessionId;
+
+        message_list.Add(next_message);
+
+        try
+        {
+            NetworkMessaging.SendJsonViaPOST(next_message);
```

Comment:
Will need to add url + route for sending message ("http://localhost:8095/sendmessagedemo")